ci/docker/manylinux.Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1.3-labs
 
 ARG HOSTTYPE
-FROM quay.io/pypa/manylinux2014_${HOSTTYPE}:2023-11-13-f6b0c51
+FROM quay.io/pypa/manylinux_2_28_${HOSTTYPE}:2024-04-23-ef7507e
```

Comment:
this is probably not going to work, as our ubuntu base image is still on 20.04 (sadly).

seems that we need to upgrade ubuntu base image first, or at the same time..

Reply:
is 20.04 that old? manylinux said that it should support linux distro back to 2018